tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Graph.java

```diff
@@ -808,7 +815,7 @@ synchronized SaverDef saverDef() {
   private SaverDef saverDef;
   private final Scope baseScope;
 
-  private final List<Op> initializers = new ArrayList<>();
+  private final Set<Operation> initializers = new LinkedHashSet<>();
```

Comment:
Do we guarantee that Operations don't have hashCode and equals implemented anywhere? This requires that we're using the identity hashcode for correct operation of the `isInitOp` check, so we might want to use `IdentityHashMap` to enforce that.

Reply:
`GraphOperation` implements `equals` and `hashCode` based on the native pointer, which is fine for this (and better than identity, if you somehow have two Java objects pointing to the same native op, they should register the same as init ops).  `EagerOperation` doesn't but should, I'll add that.

Reply:
@rnett , I think you've done that now so we can close this comment?